Source/Alamofire.swift

```diff
@@ -91,279 +95,292 @@ extension URLRequest {
     }
 }
 
-// MARK: - Request Methods
-
-/**
-    Creates a request using the shared manager instance for the specified method, URL string, parameters, and
-    parameter encoding.
-
-    - parameter method:     The HTTP method.
-    - parameter URLString:  The URL string.
-    - parameter parameters: The parameters. `nil` by default.
-    - parameter encoding:   The parameter encoding. `.URL` by default.
-    - parameter headers:    The HTTP headers. `nil` by default.
-
-    - returns: The created request.
-*/
+// MARK: - Data Request
+
+/// Creates a data `Request` using the default session manager instance to retrieve the contents of the specified
+/// `method`, `urlString`, `parameters`, `encoding` and `headers`.
+///
+/// - parameter urlString:  The URL string.
+/// - parameter method:     The HTTP method.
+/// - parameter parameters: The parameters. `nil` by default.
+/// - parameter encoding:   The parameter encoding. `.URL` by default.
```

Comment:
Proper casing for `.url`? I haven't looked, but if the enum hasn't been changed to lower case, it probably should be.